### DISCUSSION TITLE
collection hierarchy

### DISCUSSION THREAD

**Discussion by freeborders88** [thanhlmm/refi-app#72] [Q&A]
hi, 

I was able to add documents as collection at the root, but want to move this under user so that the documents by separated by users:

collection: user > documents user > collection tasks > documents task 

can this be done? 

thanks in advance for your reply

I love this product!

**Comment by thanhlmm** [comment]
Hi @freeborders88,
Basically, we can not move documents in Refi App now. This is on the road map.

For insert document with nested path like you describe, it's doable in RefiApp. Just create collection and create document by right click on the left panel
```
collection: user > documents user > collection tasks > documents task
```

**Reply by freeborders88** [reply]
yes, this works. thank you so much!

keep up the good work
